### PR TITLE
Update es-mdj.json (implements #558)

### PR DESCRIPTION
**Pull request by siwhitehouse**
Updated URLs as previous pages were moved without a redirect

### CODE REVIEW

**Review by kd-ods — CHANGES_REQUESTED** [review]
It doesn't look like this url http://www.mjusticia.gob.es/cs/Satellite/Portal/es/areas-tematicas/registros (which appears a few times in the entry) has been updated.

Also, the value for name.local is given as "es" whereas it should be "Ministerio de Justicia" and name.en should be "Ministry of Justice, Spain".

Could you please update the lastUpdated field too?

**Comment by siwhitehouse** [comment]
> It doesn't look like this url http://www.mjusticia.gob.es/cs/Satellite/Portal/es/areas-tematicas/registros (which appears a few times in the entry) has been updated.
> 
> Also, the value for name.local is given as "es" whereas it should be "Ministerio de Justicia" and name.en should be "Ministry of Justice, Spain".
> 
> Could you please update the lastUpdated field too?

It doesn't look as though I have included any updates to the record!

File: lists/es/es-mdj.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
     "en": "Ministerio de Justicia ",
```
Comment (kd-ods) [inline lists/es/es-mdj.json:3]:
```suggestion
    "en": "Ministry of Justice, Spain",

```

File: lists/es/es-mdj.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
     "en": "Ministerio de Justicia ",
     "local": "es"
```
Comment (kd-ods) [inline lists/es/es-mdj.json:4]:
```suggestion
    "local": "Ministerio de Justicia"

```

**Review by kd-ods — CHANGES_REQUESTED** [review]
A couple of suggested edits. Otherwise this looks ready to go.

**Comment by siwhitehouse** [comment]
@kd-ods Thank you - I've committed those two suggestions.

**Review by kd-ods — APPROVED** [review]
Great. I'll merge and update the front-end.

**Pull request merged**